src/types/misc/gene_associations.jl

```diff
@@ -2,10 +2,10 @@
 """
 $(TYPEDSIGNATURES)
 
-Parse `SBML.GeneProductAssociation` structure to the simpler GeneAssociation.
+Parse `SBML.GeneProductAssociation` structure to the simpler [`GeneAssociations`](@ref).
 The input must be (implicitly) in a positive DNF.
 """
-function parse_grr(gpa::SBML.GeneProductAssociation)::GeneAssociation
+function parse_grr(gpa::SBML.GeneProductAssociation)
```

Comment:
pls do not erase return types, they serve as a sanity check before stuff gets totally broken later in the pipeline.

Also the original GeneAssociation might be better renamed to GeneAssocDNF or something like that.